ci/docker-compose-build.yaml

```diff
@@ -0,0 +1,36 @@
+# Copyright 2019 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+version: '3.5'
+
+services:
+  tcf:
+    container_name: tcf
+    image: tcf-ci-build
+
+    build:
+      context: .
+      dockerfile: ../docker/Dockerfile.tcf-dev
+      args:
+        - http_proxy
+        - https_proxy
+        - no_proxy
+    volumes:
+      - ../:/project/TrustedComputeFramework
+    working_dir: "/project/TrustedComputeFramework/tools/build"
+    entrypoint: "bash -c \"\
+        if [ ${MAKECLEAN:-1} != 0 ]; then echo \\\"Clean build\\\" && make clean; fi \
+        && make \
```

Comment:
Why do you have make command (to build TCF) here instead of Dockerfile?

Reply:
We can build in Dockerfile as well, but this causes building docker image which builds openssl, sgxssl for every run is pretty time consuming. On the other hand, executing make command from compose file ensure that only source code is built and can re-use the docker image.